internal/querynodev2/server.go

```diff
@@ -296,6 +297,13 @@ func (node *QueryNode) InitSegcore() error {
 	if err != nil {
 		return err
 	}
+	deprecatedCacheWarmupPolicy := paramtable.Get().QueryNodeCfg.ChunkCacheWarmingUp.GetValue()
+	if deprecatedCacheWarmupPolicy == "sync" {
+		log.Warn("queryNode.cache.warmup is being deprecated, use queryNode.segcore.tieredStorage.warmup.vectorField instead.")
+		log.Warn("for now, if queryNode.cache.warmup is set to sync, it will override queryNode.segcore.tieredStorage.warmup.vectorField to sync.")
+		log.Warn("otherwise, queryNode.cache.warmup will be ignored")
+		vectorFieldCacheWarmupPolicy = C.CacheWarmupPolicy_Sync
+	}
```

Comment:
what if legacy yaml file contained value `async`?

Reply:
maybe we need add a warning log for this behavior

Reply:
btw, it looks like the old configuration may overwrite new one. its this the expected behavior?

Reply:
if the old is set to sync, it'll overwrite the new one